docs/advanced/security.rst

```diff
@@ -0,0 +1,52 @@
+=================================
+Cluster Security & Authentication
+=================================
+
+A Flocker cluster comprises a control service and convergence agents, along with some command line tools that are provided to interact with and manage the cluster (see :doc:`architecture`).
+
+Flocker uses `Transport Layer Security <http://en.wikipedia.org/wiki/Transport_Layer_Security>`_ (TLS) to authenticate components of a cluster against each other, using a technique known as client certification.
```

Comment:
This is usually called mutual authentication or 2-way authentication.  I've never heard the word "certification" used in TLS (as opposed to "certificates").